File: app/src/main/java/khorunzhyicom/alex/theposter/presentation/popular/PopularMoviesView.java
package khorunzhyicom.alex.theposter.presentation.popular;

import java.util.List;

import khorunzhyicom.alex.theposter.di.components.injector.PresentationInjector;
import khorunzhyicom.alex.theposter.presentation.common.view.DaggerView;
import khorunzhyicom.alex.theposter.presentation.common.view.PosterView;
import khorunzhyicom.alex.theposter.service.models.Movie;

public interface PopularMoviesView extends PosterView, DaggerView<PresentationInjector> {

    void updateMovies(List<Movie> movies);

    void showError(String error);
}

File: app/src/main/java/khorunzhyicom/alex/theposter/presentation/popular/PopularMoviesView.kt
package khorunzhyicom.alex.theposter.presentation.popular

import khorunzhyicom.alex.theposter.presentation.activity.adapter.TabView
import khorunzhyicom.alex.theposter.presentation.common.view.ContentView
import khorunzhyicom.alex.theposter.presentation.common.view.PosterBaseView
import khorunzhyicom.alex.theposter.service.models.Movie

interface PopularMoviesView : PosterBaseView, TabView, ContentView<List<Movie>>
